### PR TITLE
Use python driver metadata api

### PR DESCRIPTION
**Pull request by ptnapoleon**
Removes the existing FIXME. Review, but don't merge. I'm checking with Aleksey to make sure I'm fixing this the way he wanted before I do the rest of the tests. It seems wrong since it's more verbose, and I'm still specifying the keyspace/table names.

### CODE REVIEW

**Comment by ptnapoleon** [comment]
@mambocab This is ready for review now.

File: cqlsh_tests/cqlsh_tests.py [hunk 1]
```diff
@@ -1289,14 +1289,7 @@ def test_drop_table(self):
...
 
-        # FIXME: use python-driver metadata API
-        if self.cluster.version() >= '3.0':
-            query = "SELECT table_name FROM system_schema.tables WHERE keyspace_name='ks';"
-        else:
-            query = "SELECT columnfamily_name FROM system.schema_columnfamilies WHERE keyspace_name='ks';"
-
-        result = rows_to_list(self.cursor.execute(query))
-        self.assertEqual([], result)
+        self.assertEqual({}, self.cursor.cluster.metadata.keyspaces['ks'].tables)
```
Comment (mambocab) [inline cqlsh_tests/cqlsh_tests.py]: I don't care that this is a dictionary. I'd rather do

```
self.assertFalse(self.cursor.cluster.metadata.keyspaces['ks'].tables)
```
 
or

```
self.assertEqual(0, len(self.cursor.cluster.metadata.keyspaces['ks'].tables))
```

File: configuration_test.py [hunk 2]
```diff
@@ -93,23 +93,15 @@ def new_commitlog_cluster_node():
...
-        rows = cursor.execute(describe_table_query)
-        results = rows[0]
+        result = cursor.cluster.metadata.keyspaces['ks'].tables['test_table'].as_cql_query()
         # Now extract the param list
         params = ''
-        for result in results:
-            if 'sstable_compression' in str(result):
-                params = result
+        if 'sstable_compression' in result:
+            params = result
```
Comment (mambocab) [inline configuration_test.py]: Heck yes. I really like how you've condensed this.

File: cqlsh_tests/cqlsh_tests.py (hunk 1 above)
Comment (ptnapoleon) [inline cqlsh_tests/cqlsh_tests.py]: The latter is better, and more consistent with how I do it elsewhere. I think I only did it like this to match the old code.

**Comment by mambocab** [comment]
I'm running the changed tests now locally; I'll report back when they're done. In terms of style stuff, though, looks great. Make Travis happy and I'm happy.

**Comment by ptnapoleon** [comment]
Travis is now happy. ST3 was indenting silly. Tests pass locally for me. I'll wait for you to repro, then merge.

**Comment by mambocab** [comment]
There are a few failures when I run this branch:

https://gist.github.com/mambocab/4d2eb751c97cc989a533

but I don't think any are related to these changes. If you agree, feel free to merge.

**Pull request merged**